pom.xml

```diff
@@ -15,7 +15,7 @@
 		<jdk.version>1.8</jdk.version>
 		<!-- When updating one spring version, make sure that all of them are updated to their latest compatible versions -->
 		<spring.boot.version>2.7.4</spring.boot.version>
-		<spring.framework.version>5.3.23</spring.framework.version>
+		<spring.framework.version>4.3.29.RELEASE</spring.framework.version>
```

Comment:
Why are you downgrading the spring framework version all the way back to version 4; this is just wrong!